src/middleware/ip-restriction/index.test.ts

```diff
@@ -0,0 +1,86 @@
+import { Hono } from '../../hono'
+import { Context } from '../../context'
+import { HonoRequest } from '../../request'
+import type { GetConnInfo } from '../../helper/conninfo'
+import { ipRestriction, isMatchForRule } from '.'
+
+describe('ipLimit middleware', () => {
```

Comment:
```suggestion
describe('ipRestriction middleware', () => {
```

we should also replace s/ipLimit/ipRestriction/

Reply:
Thank you! I fixed it.

Reply:
👍